device/Microsoft.Azure.Devices.Client.NetStandard/Microsoft.Azure.Devices.Client.NetStandard.csproj

```diff
@@ -50,6 +50,10 @@
     <DocumentationFile>bin\Release_Delay_Sign\Microsoft.Azure.Devices.Client.XML</DocumentationFile>
   </PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
where is this used?

Reply:
It's not currently used but I'd rather not remove it since I'll need it for the next round of build enhancements.